packages/app-page-builder/src/types.ts

```diff
@@ -180,12 +180,12 @@ export type PbElementDataType = {
     blockId?: string;
     variables?: PbBlockVariable[];
     action?: {
-        href: string;
-        newTab: boolean;
-        clickHandler: string;
+        href?: string;
+        newTab?: boolean;
```

Comment:
Should the same changes be applied in `packages/app-page-builder-elements/src/renderers/button.tsx:80` ?